lib/turbo_tests/cli.rb

```diff
@@ -46,6 +47,10 @@ def run
           }
         end
 
+        opts.on("-o", "--test-options '[OPTIONS]'", "execute test commands with those options") do |arg| 
+          test_options << arg.strip
+        end
```

Comment:
Thanks :+1: What if `--` can be also used for test options similar to `parallel_tests`: https://github.com/grosser/parallel_tests/blob/b85a17ebd9cf920b2052623b28f0839bc35885de/lib/parallel_tests/cli.rb#L322-L339?

Reply:
heya I updated the PR body w/ more context around this.  tldr: I think it would be good for future PRs 😁